app/controllers/application_controller/authentication_dependency.rb

```diff
@@ -20,7 +20,10 @@ def adequate_scopes?
   end
 
   def authenticate_user!
```

Comment:
Might be worth adding a comment here explaining why this is our authentication logic.